### DISCUSSION TITLE
Working with translatable content in large websites (import/export?)

### DISCUSSION THREAD

**Discussion by wautersj** [statamic/cms#7580] [Q&A]
Hi all, I have a question regarding how others are using Statamic as a CMS for a large multilingual content website.

Let's say you have a few collections: 
- Pages (with several new fields inside the blueprint)
- Products (with several new fields inside the blueprint)
- Blog (with several new fields inside the blueprint)

And the content creator finished the entire website in a certain language (let's say: English)
All pages, products and blogs are accounted for.

Now the company wants the site translated in another language (let's say: German)
The company wants to 'export the entire website', to send it to our localization partner, that will be in change of translating all the content from English to German.
The localization partner does not want to work directly inside the CMS. They probably have software to help replace/find large bits of text inside delivery files (in a certain format) to help speed up their translation process.

I have no knowledge of a way to 'export' the entire content with the values of the standaard (ootb) fields as well as the extra added fields in the blueprints. (So that another company can work inside those files and later on let us 'import' them again as the new language.) 

I've tried looking for import/export addons; but no success. I understand that working with extra fields/bards/replicators/... somewhat makes it more difficult if not impossible to get to a sort of statamic standard of serializing and deserializing content files.

I've tried to copy and paste the entire "EN" folder to a "DE" folder, and see if the site now technically still worked (now in DE as well) but no; A) since the translated entries use "origin" to link to their origin file in EN, .... B) even if I were to change the property in the file from "id" to "origin", the DE file must still have a "id" property that is used in the 'trees' files, and so on ... C) not all properties in the EN file may be presented in the DE file; some properties stay in the origin file.

I don't really know where to go from here. I can't find much about this matter online. 

What are your experiences with handeling translated content in bullk?

**Comment by edalzell** [comment]
Ask them what format they need it in.

If it's search replaces, give them a zip of the content, they can use that.
